operator_csv_libs/csv.py

```diff
@@ -63,11 +63,19 @@ def set_deployments_annotations(self, key=None, value=None):
                 continue
             d['spec']['template']['metadata']['annotations'][key] = value
 
+    def set_container_image(self, image):
```

Comment:
I wonder if a method name like set_container_image_annotation would make the code clearer on the consumption side. I.e. making it easier to understand that it's not the deployment image that is manipulated, but rather a csv annotation